fit/core-reference/src/test/java/org/apache/syncope/fit/core/RecertificationITCase.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2016 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.syncope.fit.core;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import javax.ws.rs.core.Response;
+import org.apache.commons.lang3.time.DateUtils;
+import org.apache.syncope.common.lib.to.SchedTaskTO;
+import org.apache.syncope.common.lib.to.WorkflowFormPropertyTO;
+import org.apache.syncope.common.lib.to.WorkflowFormTO;
+import org.apache.syncope.common.rest.api.beans.ExecuteQuery;
+import org.apache.syncope.common.rest.api.service.TaskService;
+import org.apache.syncope.fit.core.reference.TestIdentityRecertification;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.JVM)
+public class RecertificationITCase extends AbstractTaskITCase{
+
+    private void runRecertTask() {
+        SchedTaskTO task = new SchedTaskTO();
+        task.setActive(true);
+        task.setName("IdentityRecert");
+        task.setJobDelegateClassName(TestIdentityRecertification.class.getName());
+
+        Response response = taskService.create(task);
+        task = getObject(response.getLocation(), TaskService.class, SchedTaskTO.class);
+        assertNotNull(task);
+
+        Date initial = new Date();
+        Date later = DateUtils.addSeconds(initial, 2);
+
+        taskService.execute(new ExecuteQuery.Builder().key(task.getKey()).startAt(later).build());

```

Comment:
Any particular reason for not executing the task via `AbstractTaskITCase#execTask`?